SimpleWAWS/Code/BackgroundQueueManager.cs

```diff
@@ -61,26 +63,31 @@ public void DeleteResourceGroup(ResourceGroup resourceGroup)
 
         private async Task<ResourceGroup> LogActiveUsageStatistics(ResourceGroup resourceGroup)
         {
-            try
+            if (resourceGroup.SubscriptionType == Subscription.SubscriptionType.AppService)
             {
-                var site = resourceGroup.Sites.First(s => s.IsSimpleWAWSOriginalSite);
-                var credentials = new NetworkCredential(site.PublishingUserName, site.PublishingPassword);
-                var zipManager = new RemoteZipManager(site.ScmUrl + "zip/", credentials);
-                using (var httpContentStream = await zipManager.GetZipFileStreamAsync("LogFiles/http/RawLogs"))
+                try
                 {
-                    await StorageHelper.UploadBlob(resourceGroup.ResourceUniqueId, httpContentStream);
+                    foreach (var site in resourceGroup.Sites)
```

Comment:
I am appending the sitename below to the unique id for now
`var uniqueId = String.Concat(resourceGroup.ResourceUniqueId, '_', site.SiteName); 
`
I'll check the `resourcetype` and only upload the relevant site logs with the next commit  before merging.